src/lib/services/index.ts

```diff
@@ -165,6 +165,18 @@ export const scheduleServices = async (
         'removeInstancesOlderThanTwoDays',
     );
 
+    schedulerService.schedule(
+        clientInstanceService.bulkAdd.bind(clientInstanceService),
+        secondsToMilliseconds(5),
+        'bulkAddInstances',
+    );
+
+    schedulerService.schedule(
+        clientInstanceService.announceUnannounced.bind(clientInstanceService),
+        minutesToMilliseconds(5),
+        'announceUnannounced',
+    );
```

Comment:
Where these previously configurable by environment variables? Or were the parameters in instance-service.ts never used?

Reply:
They were default parameters on the constructor that never got overwritten in prod, only in tests